themes/climate.json

```diff
@@ -19,6 +18,18 @@
   "--footer--copyright--year--color": "#66cc00",
   "--footer--links--color": "#66cc00",
   "--footer--links--hover--color": "#66cc00",
+  "--footer--background": "#ffffff",
```

Comment:
These are the new variables added to new footer design

Reply:
Keep in mind we also need to update the other 2 themes. But probably it's going to be easier to do this last, in case we change names so you don't have to keep updating these 4 files.

Reply:
Got it! I'll keep this conversation open, then.